transit_odp/timetables/loaders.py

```diff
@@ -200,21 +200,24 @@ def load_services(self, revision):
 
     def load_vehicle_journeys(self, service_patterns):
         vehicle_journeys = self.transformed.vehicle_journeys
-
         if not vehicle_journeys.empty:
             if not service_patterns.empty:
                 service_patterns = (
                     service_patterns.reset_index()[
-                        ["service_pattern_id", "id", "file_id"]
+                        ["service_pattern_id", "id", "file_id", "line_name"]
                     ]
                     .drop_duplicates()
                     .rename(columns={"id": "id_service"})
                 )
+                # add line_name to the VJs
+                vehicle_journeys["line_name"] = (
+                    vehicle_journeys["line_ref"].str.split(":").str[-1]
```

Comment:
Please verify that line_ref will always have a value and of the expected format. str[-1] might throw an exception if semi colon is not found in the string

When possible it might be worth to merge lines to obtain line names